lib/bundler_api/gem_info.rb

```diff
@@ -16,19 +16,21 @@ def deps_for(gems = [])
       if gems.any?
         @conn[<<-SQL, Sequel.value_list(gems)]
           SELECT rv.name, rv.number, rv.platform, rv.required_ruby_version, rv.checksum,
-                 rv.rubygems_version, d.requirements, for_dep_name.name dep_name
+                 rv.rubygems_version, d.requirements, rv.created_at, for_dep_name.name dep_name
           FROM
             (SELECT r.name, v.number, v.platform,v.rubygems_version, v.checksum,
-                    v.required_ruby_version, v.id AS version_id
+                    v.required_ruby_version, v.created_at, v.id AS version_id
             FROM rubygems AS r, versions AS v
             WHERE v.rubygem_id = r.id
               AND v.indexed is true
-              AND r.name IN ?) AS rv
+              AND r.name IN ?
+              ORDER BY v.created_at, v.number, v.platform) AS rv
           LEFT JOIN dependencies AS d ON
             d.version_id = rv.version_id
           LEFT JOIN rubygems AS for_dep_name ON
             d.rubygem_id = for_dep_name.id
-            AND d.scope = 'runtime';
+            AND d.scope = 'runtime'
+          ORDER BY rv.created_at, rv.number, rv.platform;
```

Comment:
Actually I think it is not good enough it and it worked by luck. This last order by should include `for_dep_name.name`. Right?